### PR TITLE
0.11.0

### LINKED ISSUE #43: Add comparison info from main site to overview docs

### LINKED ISSUE #50: reintroduce 'lastmodified' to pages
Something flaked with merging to main so I've temporarily removed it, reintroduce it.

### LINKED ISSUE #56: Update node status UI links
Update the nodesguru link + add nymspy link here: https://nymtech.net/docs/run-nym-nodes/mixnodes/#checking-that-your-node-is-mixing-correctly 

Also do a small reword.

### LINKED ISSUE #58: static formatting for port reference tables
port tables currently default to the entire page width - change this so they are static / columns are at most the size of the widest element

### PR DESCRIPTION
**Pull request by mfahampshire**
PR for `v0.11.0` documentation, including updates where necessary, optimized setup guide commands, and port references.

### CODE REVIEW

File: content/build-apps/running-localnet.draft [hunk 1]
```diff
@@ -12,9 +13,9 @@ You can start up a full mix network, including mixnodes, and a gateway on your l
 
 First, download, build and run the [Nym directory server](https://github.com/nymtech/nym-directory). This step will soon go away, as the directory server is about to be retired in the {{< param stable >}}  release. But for now it's necessary.
 
-{{% notice warning %}}
+{{< attention title=" " >}}
 The local network **must** be able to access a directory server running on `localhost`! Make sure you install and run the directory server.
```
Comment (futurechimp) [inline content/build-apps/running-localnet.draft]: There's no directory server anymore (users would need to run a valdiator and valdiator api). Probably the only practical way to get a full local setup running in under a few hours would be to use Bogdan's Docker containers. However I'd be reluctant to point people at those in the docs until we see how brittle they turn out to be. 
For the moment I would just remove this whole section until we get it right.

Comment (mfahampshire) [reply content/build-apps/running-localnet.draft]: this is a leftover draft from before i joined the team - I can just remove this page entirely if you want?

File: content/overview/nym-platform.md [hunk 2]
```diff
@@ -3,8 +3,9 @@ title: The Nym Platform
 weight: 10
 description: "An overview of the Nym platform architecture"
 ---
+{{< lastmodified >}}
 
-We are currently running a [testnet](https://explorer.nymtech.net) with Nym validators and mixnodes. Together, the validators and mixnodes provide integrated access control and network privacy to users of the Nym platform. They are assisted by several other platform components: multiple types of clients, and gateway nodes.
+We are currently running a [testnet](https://testnet-finney-explorer.nymtech.net/) with Nym validators and mixnodes. Together, the validators and mixnodes provide integrated access control and network privacy to users of the Nym platform. They are assisted by several other platform components: multiple types of clients, and gateway nodes.
```
Comment (futurechimp) [inline content/overview/nym-platform.md]: I guess we would want to update this one to milhon?

Comment (mfahampshire) [reply content/overview/nym-platform.md]: yep missed that - fixing now

**Pull request merged**